int32 cameraID
std_msgs/Header header
Object[] objects